src/datadog/azext_datadog/generated/_params.py

```diff
@@ -0,0 +1,183 @@
+        c.argument('created_by', type=str, help='The user that created the API key.')
+        c.argument('name', type=str, help='The name of the API key.')
+        c.argument('key', type=str, help='The value of the API key.')
+        c.argument('created', type=str, help='The time of creation of the API key.')
+
+    with self.argument_context('datadog host list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('monitor_name', type=str, help='Monitor resource name')
+
+    with self.argument_context('datadog linked-resource list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('monitor_name', type=str, help='Monitor resource name')
+
+    with self.argument_context('datadog monitored-resource list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('monitor_name', type=str, help='Monitor resource name')
+
+    with self.argument_context('datadog monitor list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('datadog monitor show') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('monitor_name', options_list=['--name', '-n', '--monitor-name'], type=str, help='Monitor resource '
+                   'name', id_part='name')
+
+    with self.argument_context('datadog monitor create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('monitor_name', options_list=['--name', '-n', '--monitor-name'], type=str, help='Monitor resource '
+                   'name')
+        c.argument('tags', tags_type)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx),
+                   validator=get_default_location_from_resource_group)
+        c.argument('identity_type', arg_type=get_enum_type(['SystemAssigned', 'UserAssigned']), help='Identity type')
+        c.argument('provisioning_state', arg_type=get_enum_type(['Accepted', 'Creating', 'Updating', 'Deleting', ''
+                                                                 'Succeeded', 'Failed', 'Canceled', 'Deleted', ''
+                                                                 'NotSpecified']), help='Provisioning state')
+        c.argument('monitoring_status', arg_type=get_enum_type(['Enabled', 'Disabled']), help='Flag specifying if the '
+                   'resource monitoring is enabled or disabled.')
+        c.argument('marketplace_subscription_status', arg_type=get_enum_type(['Active', 'Suspended']), help='Flag '
+                   'specifying the Marketplace Subscription Status of the resource. If payment is not made in time, '
+                   'the resource will go in Suspended state.')
+        c.argument('datadog_organization_properties', action=AddDatadogOrganizationProperties, nargs='*', help='')
+        c.argument('user_info', action=AddUserInfo, nargs='*', help='')
```

Comment:
Please add help message for these 2 properties.

Reply:
Updated.